SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -103,3 +105,13 @@ bool UGridBasedLBStrategy::IsInside(const FBox2D& Box, const FVector2D& Location
 	return Location.X >= Box.Min.X && Location.Y >= Box.Min.Y
 		&& Location.X < Box.Max.X && Location.Y < Box.Max.Y;
 }
+
+UGridBasedLBStrategy::LBStrategyRegions UGridBasedLBStrategy::GetLBStrategyRegions() const
+{
+	LBStrategyRegions VirtualWorkerToCell;
```

Comment:
Nit: could pre-allocate the memory you need here. We know we are going to use WorkerCells.Num() elements.

Don't have to do this...